src/main/flight/mixer.h

```diff
@@ -66,6 +66,7 @@ typedef struct mixer_t {
 typedef struct mixerConfig_s {
     uint8_t pid_at_min_throttle;            // when enabled pids are used at minimum throttle
     int8_t yaw_direction;
+    int8_t yaw_jump_prevention_limit;       // make limit configurable (original fixed value was 100)
```

Comment:
Original value is 100. i set the maximum in the GUI to 250. this should cover quite some range?